docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -227,7 +235,7 @@ import javax.enterprise.context.ApplicationScoped;
 public class GreetingService {
 
     public String greeting(String name) {
-        return "hello " + name + "\n";
+        return "hello " + name;
```

Comment:
We need the corresponding quickstart to be aligned with this if it's not already the case.

Reply:
all the quickstarts I could find does not have \n. That is something only added to the getting started guide and one of the reasons I suggested to roll it back since it makes things "different" and quirky as it breaks the tests until you have updated them in the future steps.

Reply:
I was thinking about this code https://github.com/quarkusio/quarkus-quickstarts/blob/master/getting-started/src/main/java/org/acme/quickstart/GreetingResource.java#L27 but apparently it was not in sync. Looks good then.